src/ontology/mondo-edit.obo

```diff
@@ -537488,6 +537488,7 @@ synonym: "GARD:0000632" EXACT [MONDO:equivalentTo]
 xref: MEDGEN:82914 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: UMLS:C0276496 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:82914"}
 is_a: MONDO:0004975 {source="https://orcid.org/0000-0001-5208-3432"} ! Alzheimer disease
+is_a: MONDO:0024237 {source="https://orcid.org/0000-0002-0587-4693"} ! inherited neurodegenerative disorder
```

Comment:
@yshwetar can you add another source for this parent? maybe a PMID? or maybe one of the xref that mention it? It is good to add an actual record for the source (in addition to only an orcid) when possible, and I think this shouldn't be too difficult to find. Thank you!

Reply:
Comment addressed